public/app/plugins/datasource/zipkin/QueryField.test.tsx

```diff
@@ -20,8 +20,8 @@ describe('QueryField', () => {
     );
 
     expect(wrapper.find(ButtonCascader).length).toBe(1);
-    expect(wrapper.find('input').length).toBe(1);
-    expect(wrapper.find('input').props().value).toBe('1234');
+    expect(wrapper.find(QueryField).length).toBe(1);
+    expect(wrapper.find(QueryField).prop('query')).toBe('1234');
```

Comment:
1 more thing not directly related to your PR - in this file on line 55 we have forgotten console.log `console.log({ url });` . We could remove it.